pkgs/applications/editors/hmm/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, makeWrapper
+, php
+, ncurses
+, coreutils
+, xclip
+}:
```

```suggestion
, xclip
, fetchFromGitHub
}:
```
You would ofc also need this line to use fetchFromGithub if you choose that route